diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/identifiers/LocalVariablesRule.kt

```diff
@@ -94,12 +96,28 @@ class LocalVariablesRule(private val configRules: List<RulesConfig>) : Rule("loc
     }
 
     private fun handleConsecutiveDeclarations(statement: PsiElement, properties: List<KtProperty>) {
+        val numLinesAfterLastProp = if (properties.last().node.treeNext.elementType == WHITE_SPACE) {
+            // minus one is needed to except \n after property
+            properties
+                    .last()
+                    .node
+                    .treeNext
+                    .numNewLines() - 1
+        } else {
+            0
+        }
+
         // need to check that properties are declared consecutively with only maybe empty lines
         properties
             .sortedBy { it.node.getLineNumber() }
-            .zip(properties.size - 1 downTo 0)
-            .forEach { (property, offset) ->
-                checkLineNumbers(property, statement.node.getLineNumber(), offset)
+            .let { it as MutableList }
+            .zip(((properties.size - 1) downTo 0).map { it + numLinesAfterLastProp })
+            .forEachIndexed { index, (property, offset) ->
+                if (index != properties.lastIndex) {
+                    checkLineNumbers(property, statement.node.getLineNumber(), offset)
+                } else {
+                    checkLineNumbers(property, statement.node.getLineNumber(), 0)
```

Comment:
So, you increase offset by number of empty lines after the last property, which sort of makes sense. But then you ignore this adjustment for this last property. What exactly is going on here? Please add some more comments in code.